sda-auth/main.go

```diff
@@ -417,6 +420,15 @@ func main() {
 	app.Get("/elixir/login", authHandler.getElixirLogin)
 	app.Get("/elixir/cors_login", authHandler.getElixirCORSLogin)
 
+	publicKey, err := readPublicKeyFile(authHandler.Config.PublicFile)
+	if err != nil {
+		log.Fatalf("Failure to get public key: %s", err.Error())
+	}
+	authHandler.pubKey = hex.EncodeToString(publicKey[:])
```

Comment:
I would place this above line 378 so that everything is configured and ready before the iris app gets initiated.
Then the `hex.EncodeToString` can be done directly in the `authHandler` struct declaration.